roles/opstoolsvhost/templates/opstools.conf.j2

```diff
@@ -0,0 +1,28 @@
+{% if opstools_apache_http_port|int != 80 %}
+Listen {{ opstools_apache_http_port }}
+{% endif %}
+{% if opstools_apache_https_port|int != 443 %}
+Listen {{ opstools_apache_https_port }} https
+{% endif %}
+
+<VirtualHost *:{{ opstools_apache_http_port }}>
+  RewriteEngine On
+  RewriteCond %{HTTPS} off
+  RewriteRule (.*) https://%{HTTP_HOST}$1 [R,L]
+</VirtualHost>
+
+<VirtualHost *:{{ opstools_apache_https_port }}>
+  RewriteEngine On
+  ProxyRequests Off
+  SSLEngine On
```

Comment:
one more: package mod_ssl needs to be installed. I tested with a fresh and minimal CentOS 7 image